src/olympia/addons/forms.py

```diff
@@ -156,10 +156,11 @@ class AddonFormBasic(AddonFormBase):
     summary = TransField(widget=TransTextarea(attrs={'rows': 4}),
                          max_length=250)
     tags = forms.CharField(required=False)
+    is_experimental = forms.BooleanField(initial=False, required=False)
```

Comment:
The `initial=False` here is suspicious. Shouldn't it follow the model instance value instead, whatever that currently is ?